keras_hub/src/utils/transformers/export_gemma_to_safetensor.py

```diff
@@ -0,0 +1,134 @@
+import json
+import os
+
+import torch
+from safetensors.torch import save_file
```

Comment:
does this work on all backends? or do we need to flip between versions depending on the backend? worth testing out